### PR TITLE
Added reana file and partial instructions/ Fixed contributors data

### PR DESCRIPTION
**Pull request by diyaselis**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -188,11 +188,75 @@ Using CWL:
 
     $ cwltool --outdir=./outputs ./workflow/workflow.cwl ./workflow/input.yaml
 
+Create REANA file
+=================
```
Comment (tiborsimko) [inline README.rst]: We now use a different wording with the release of REANA v0.3.0, please see the example https://github.com/reanahub/reana-demo-worldpopulation/  Could you amend the wording here as well?

File: README.rst [hunk 2]
```diff
@@ -188,11 +188,75 @@ Using CWL:
...
+.. code-block:: yaml
+   
+    version: 0.2.0
+    metadata: 
+      authors:
+       - Diyaselis Delgado Lopez <diyaselis.delgado@upr.edu>
+      title: Higgs to 4 lepton physics analysis example
+      date: 22 September 2018
+      repository: https://github.com/reanahub/reana-demo-cms-h4l/
+    code:
```
Comment (tiborsimko) [inline README.rst]: ... also the structure of `reana.yaml` was changed with the v0.3.0, e.g. there is now no distinction between "code" and "data" inputs in the YAML file. Can you please emulate the example from  https://github.com/reanahub/reana-demo-worldpopulation/  regarding YAML file and the running instructions?

**Comment by diyaselis** [comment]
@tiborsimko Changed the reana.yaml, please revise the command-line rundown. Also inputs folder to be switch to data?

**Comment by tiborsimko** [comment]
@diyaselis Did you close this PR by accident or on purpose? (i.e. will there be another one?)

**Comment by diyaselis** [comment]
@tiborsimko I need to review the reana file, workflows and instructions, then I'll create a new PR.

**Pull request closed without merging**